### PR TITLE
Update to new config artifact/version scheme

### PR DESCRIPTION
**Pull request by kenfinnigan**

### CODE REVIEW

**Comment by kenfinnigan** [comment]
5 tests failed locally, but it appears they've been failing for a while

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
   <properties>
     <version.microprofile.opentracing>1.3.1</version.microprofile.opentracing>
     <version.microprofile.config>1.3</version.microprofile.config>
-    <version.smallrye.config>1.3.5</version.smallrye.config>
+    <version.smallrye.config>1.0.0</version.smallrye.config>
     <version.smallrye.rest.client>1.2.1</version.smallrye.rest.client>
```
Comment (pavolloffay) [inline pom.xml]: Not updating the rest-client?

**Comment by pavolloffay** [comment]
It seems that CI passed here. TCK seems to be flaky see #54

File: pom.xml (hunk 1 above)
Comment (Ladicek) [reply pom.xml]: RestClient moved to RESTEasy.

**Comment by kenfinnigan** [comment]
You good with this @pavolloffay?

I've asked the RESTEasy guys when we can have a release to update REST Client here

**Comment by pavolloffay** [comment]
Yes it looks good me.

**Pull request merged**